### ISSUE TITLE
Support fixed with for table icon columns if reasonable

### ISSUE THREAD

**Issue by m-akinc** [ni/nimble#1522]
## 🧹 Tech Debt

The table icon column currently supports fractional width/sizing. An icon column really seems better suited to a fixed width, since icons are a standard size. However, the minimum column width is quite large if you reserve space for all the elements that appear in the header. We need to determine whether there is some better way to support fixed widths for icon columns.

Note that there is another issue #1515 that requests support for more than just a single icon in an icon column. If we generalize to supporting text + icon(s), it could affect how we approach this issue.

**Comment by jattasNI** [comment]
@m-akinc is this the same as #1332? Or a subset of it?

**Comment by m-akinc** [comment]
Let's treat this as the work to update the icon column specifically after #1332 generally addresses the issue of fixed-width columns.
